lib/rubocop/cop/style/trailing_comma.rb

```diff
@@ -92,11 +90,16 @@ def check(node, items, kind, begin_pos, end_pos)
               avoid_comma(kind, after_last_item.begin_pos + comma_offset, sb,
                           extra_info)
             end
-          elsif should_have_comma
+          elsif should_have_comma?(style, node, items)
             put_comma(items, kind, sb)
           end
         end
-        # rubocop:enable Metrics/MethodLength
+
+        def should_have_comma?(style, node, items)
+          [:comma, :consistent_comma].include?(style) &&
+            multiline?(node) &&
+            (items.size > 1 || items.last.hash_type?)
```

Comment:
Not sure if `items.size > 1` should be applied to Hashes. Currently rubocop forces me to do:

```ruby
{
  'data' => [],
  'meta' => {
    'pagination' => {
      'offset' => offset,
      'count' => count,
      'limit' => limit,
      'total' => total,
    }
  },
}
```

instead of:

```ruby
{
  'data' => [],
  'meta' => {
    'pagination' => {
      'offset' => offset,
      'count' => count,
      'limit' => limit,
      'total' => total,
    },
  },
}
```